packages/amplify-category-analytics/src/index.ts

```diff
@@ -30,16 +29,7 @@ export const console = async (context: $TSContext): Promise<void> => {
 
   let selectedResource;
   if (hasKinesisResource && hasPinpointResource) {
-    const question = {
-      name: 'resource',
-      message: 'Select resource',
-      type: 'list',
-      choices: ['kinesis', 'pinpoint'],
-      required: true,
-    };
-
-    const result = await inquirer.prompt(question as QuestionCollection<{ [x: string]: unknown }>);
-    selectedResource = result.resource;
+    selectedResource = await prompter.pick('Select resource', ['kinesis', 'pinpoint']);
```

Comment:
I am not sure what required means here? if we didn't allow empty selection before we might need to set the `pickAtLeast: 1` option. Can you please confirm this?

Reply:
When testing I didn't see an obvious way to bypass this selection. I checked the prompter and inquirer and the default behavior is to require a choice when prompted in the pick one scenario.